polaris-react/src/components/Box/Box.tsx

```diff
@@ -121,6 +61,13 @@ export type BorderRadiusTokenScale =
   | 'large'
   | 'half';
 
+export type BackgroundColors =
+  | ColorsBackdropTokenAlias
+  | ColorsBackgroundTokenAlias
+  | ColorsOverlayTokenAlias
+  | ColorsActionTokenAlias
+  | ColorsSurfaceTokenAlias;
```

Comment:
Thoughts on this group being exported from tokens as well? Not sure if we'll need it elsewhere